javascript/node/selenium-webdriver/package.json

```diff
@@ -36,7 +36,7 @@
     "sinon": "^7.5.0"
   },
   "scripts": {
-    "test": "mocha -t 600000 --recursive test",
+    "test": "mocha -t 600000 --recursive ./**/*_test.js",
```

Comment:
What glob did you run to test that the same files are being pulled by mocha?

Reply:
will back to initial